__tests__/endpoints.test.js

```diff
@@ -143,6 +143,82 @@ describe("/api/articles/:article_id/comments", () => {
   });
 });
 
+describe("POST: /api/articles/:article_id/comments", () => {
+  test("POST: 200 responds with the comment added for an article", () => {
+    return request(app)
+      .post("/api/articles/5/comments")
+      .send({ username: "icellusedkars", comment: "good one" })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.insertedComment).toMatchObject({
+          comment_id: expect.any(Number),
+          votes: expect.any(Number),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          author: expect.any(String),
+        });
+      });
+  });
+
+  test("check if insertion is successful", () => {
+    return request(app)
+      .post("/api/articles/5/comments")
+      .send({ username: "icellusedkars", comment: "good one" })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.insertedComment).toMatchObject({
+          comment_id: expect.any(Number),
+          votes: expect.any(Number),
+          article_id: 5,
+          body: "good one",
+          created_at: expect.any(String),
+          author: "icellusedkars",
+        });
+      });
+  });
+
+  test("POST: 400 responds with an appropriate status and error message when given an invalid id", () => {
+    return request(app)
+      .post("/api/articles/not-a-number/comments")
+      .send({ username: "icellusedkars", comment: "good one" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+
+  test("POST: 404 responds with an appropriate status and error message when given a non-existent id", () => {
+    return request(app)
+      .post("/api/articles/999/comments")
+      .send({ username: "icellusedkars", comment: "good one" })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Id Not Found(foreign key constraint violated)");
```

Comment:
This error message is a little too specific - a 404: article not found would be easier for a user to understand